hivemq-edge/src/frontend/src/extensions/datahub/designer/operation/OperationNode.tsx

```diff
@@ -25,12 +25,20 @@ export const OperationNode: FC<NodeProps<OperationData>> = (props) => {
       <NodeWrapper route={`node/${DataHubNodeType.OPERATION}/${id}`} {...props}>
         <VStack>
           <HStack w="100%" justifyContent="space-around">
-            {isSerialiser && <Text fontSize="xs">{OperationData.Handle.SCHEMA}</Text>}
+            {isSerialiser && (
+              <Text fontSize="xs">{t('workspace.handles.operation', { context: OperationData.Handle.SCHEMA })}</Text>
+            )}
             {isTransform && (
               <>
-                <Text fontSize="xs">{OperationData.Handle.DESERIALISER}</Text>
-                <Text fontSize="xs">{OperationData.Handle.FUNCTION}</Text>
-                <Text fontSize="xs">{OperationData.Handle.SERIALISER}</Text>
+                <Text fontSize="xs">
+                  {t('workspace.handles.operation', { context: OperationData.Handle.DESERIALISER })}
```

Comment:
I could not find this translation in the json files

Reply:
There are not missing: 
```
      "behavior_serial.publish": "onPublish"
      "behavior_serial.publish": "onPublish",
      "operation_deserialiser": "deserializer",
      "operation_serialiser": "serializer",
      "operation_function": "functions",
      "operation_schema": "schemas"
```
I'm using the context interpolation to avoid having to use variables in the key

Reply:
oh, this is great. didn't know about this trick.